runtime/runtime-params-estimator/estimator-warehouse/src/estimate.rs

```diff
@@ -54,8 +57,8 @@ pub(crate) fn run_estimation(db: &Db, config: &EstimateConfig) -> anyhow::Result
     let iters = 5.to_string();
     let warmup_iters = 1.to_string();
 
-    // time metric
-    {
+    let metrics: Vec<_> = config.metrics.split(',').collect();
```

Comment:
this is built into clap: https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.use_value_delimiter

Reply:
nice, didn't know that

Reply:
I didn't either, it's just that I hate how clap is a kitchen sink, so I'd be surprised if it cant' do something.